lighthouse-core/gather/driver.js

```diff
@@ -160,13 +160,13 @@ class Driver {
   }
 
   /**
-   * Computes the ULTRADUMB™ benchmark index to get a rough estimate of device class.
```

Comment:
awwww it's learning :)

Reply:
heh, yeah I figured now that there was a smidge of thought put into it and actual guidance from js-engine folks it needed a slight upgrade :)